docs/jumpstart/week-10.md

```diff
@@ -0,0 +1,291 @@
+
+```ignorelang
+web: java $JAVA_OPTS -cp target/classes:target/dependency/* <project main package>
+```
+
+In our case, if you download our catch-up code, our Procfile looks like:
+```ignorelang
+web: java $JAVA_OPTS -jar service/target/service-1.0-SNAPSHOT-jar-with-dependencies.jar
+```
+
+This Procfile specifies that our application will use the web, indicated by the "web" portion, and our "java" section specifies how to start it up. 
+The last portion of the Procfile specifies how to run our jar file. A jar file is a file used to run a Java application. You can use a jar file to run your Java code anywhere. It's easily exported and
+is how Heroku runs our application. <strong> Our Procfile is located in our project's root directory. </strong>
+
+The next configuration we need is a <strong> system.properties </strong> file. This is a file that Heroku uses to define our application's properties, such as the Java and Maven versions. This properties
+file is different from our <strong> service/.../resources/properties </strong> directory as this property file is only for Heroku.
+
+Our system.properties looks like
+```ignorelang
+java.runtime.version=11
+maven.version=3.6.2
+```
+<strong> Make sure this file is in our root directory with your Procfile so that Heroku can easily identify this file. </strong>
+
+Once we have all of our configurations set up, we can begin deploying our backend. Navigate to our backend directory and run the command:
+
+```shell
+mvn clean install
+```
+
+This will clean our application and build it again.
+Once we verified that our application successfully built, we follow the same steps as deploying our frontend code.
+
+If we have not committed our changes to the master branch make so to do so with:
+
+```shell
+git add .
+```
+which adds all our files to our project's git repository
+
+```shell
+git commit -m
+```
+which commits our added files
+
+Once we have committed our code, we now have to push it into our master branch on git. To do this we run the command:
+
+```shell
+git push <branch_name>:master
+```
+
+If you have been working from the starter code from the first week, then our branch name is called starter-code. If you have created your own
+git repository, make sure to replace <branch_name> with the name of your branch if you are <strong> not </strong> in the master branch already.
+
+
+We should already be logged into Heroku because we deployed our frontend code but if we're not, then we have to login again using:
+```shell
+heroku login
+```
+
+After logging in we run:
+
+```shell
+heroku create
+```
+which creates a new project on Heroku that represents our backend
+
+```shell
+git push heroku master
+```
+This will push our code to the Heroku server. Again, if we want to deploy a version that is not in our master branch we run the command:
+```shell
+git push heroku <branch_name>:master
+```
+This will push our designated git branch but Heroku pretends that this branch is the master branch.
```

Comment:
```suggestion
This will push our designated git branch. Again, you should merge your branch into master using the steps above if you're not on master already.
```